clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/MetadataRepoBasedTopicExistingCheckerImpl.java

```diff
@@ -26,17 +26,20 @@ public boolean checkTopicExists(String topic) {
       if (Version.isVersionTopicOrStreamReprocessingTopic(topic)) {
         int version = Version.parseVersionFromKafkaTopicName(topic);
         if (store.getVersion(version) == null) {
+          LOGGER.warn("Version {} not found for topic: {}", version, topic);
           isExistingTopic = false;
         }
       } else if (Version.isRealTimeTopic(topic)) {
         if (!store.isHybrid()) {
+          LOGGER.warn("Store {} is not hybrid currently, but found real-time topic {}", storeName, topic);
           isExistingTopic = false;
         }
       }
     } catch (VeniceNoStoreException e) {
+      LOGGER.warn("Store not found for topic: {}", topic);
       isExistingTopic = false;
     } catch (Exception e) {
-      logger.error("Exception thrown in checkTopicExists: ", e);
+      LOGGER.error("Exception thrown in checkTopicExists: ", e);
     }
     return isExistingTopic;
```

Comment:
This is a very minor nitpick, but IMHO the code would be cleaner if we deleted this `isExistingTopic` variable and just returned true here, and returned false directly in all the cases above where we set the variable to false.

Reply:
Updated.